### DISCUSSION TITLE
Timing for web support

### DISCUSSION THREAD

**Discussion by ethan-drivr** [isar/isar#786] [Help] [answered]
I know it takes a big and tireless commitment to build a quality open-source project like this. Thank you so much for doing this. You have built something amazing that I think will continue to grow in popularity over time. 

I'm part of a little start-up and we are working really hard to make some big dreams come true. Isar is a key part of our flutter implementation and we love it. But now I'm coming to a cross-roads. I need to implement web support soon and I don't have any idea at the moment when it will be available for isar. Originally isar's web support was one of the factors that influenced our decision to use it.

**Would you be able to give some idea of the timing for web support? Even a general idea would be super helpful at this point.**

(I've attempted to back-port my codebase to 2.5 for web support and have encountered multiple issues. At this point I'm not confident it's a viable option.)

**Comment by simc** [comment, marked as answer]
My goal is to support it this year

**Reply by ethan-drivr** [reply]
Thanks for the update, I appreciate it! I'll go ahead an open a separate discussion to see if you can help me with the issue I'm seeing with 2.5.

**Comment by cgestes** [comment]
@leisim can we have an overview of the issues? I was wondering if we could have some basic functionality working before the official support.

Thanks a lot :)

**Comment by gustavorozolin** [comment]
@leisim what if you release a version as a stopgap with the same technology used in version 2.5?

**Comment by vogttho** [comment]
Please remove web support from pudbdev description.
